src/main/kotlin/pt/up/fe/ni/website/backend/service/AccountService.kt

```diff
@@ -35,7 +52,13 @@ class AccountService(private val repository: AccountRepository, private val enco
             throw IllegalArgumentException(ErrorMessages.emailAlreadyExists)
         }
 
+        dto.photoFile?.let {
+            val fileName = photoFilename(dto.email, it)
+            account.photo = fileUploader.uploadImage("profile", fileName, it.bytes)
+        }
+
         val newAccount = dto.update(account)
```

Comment:
Is the image kept if not included on `UpdateAccountDto`? I assume it is but wanted to know if you've tested

Reply:
I have a test for this: src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt#749